doc/design/quic-programming-guide.md

```diff
@@ -63,33 +63,33 @@ The server API calling flow is shown in below diagram and table.
-| 3 | Application Creates the Visitor instance of QuicTransportServerInterface |
-| 4 | Application calls SetVisitor on the QuicTransportServerInterface instance got in step #2. |
-| 5 | Application calls Start() method on the QuicTransportServerInterface instance got in step #2 |
-| 6 | OnSession() callback will be invoked once the quictransportserver gets a connection request from client, and an QuicTransportSession instance is passed from the callback. |
-| 7 | Application calls CreateBidirectionalStream on the QuicTransportSession got in step 6 and get a QuicTransportStreamInterface instance. |
-| 8 | Application creates QuicTransportStream::Visitor instance and invokes QuicTransportStreamInterface::SetVisitor(). |
-| 9 | Application reads the QuicTransportStream when OnCanRead is invoked on the QuicTransportStream::Visitor; or write to the QuicTransportStream when OnCanWrite is invoked on the QuicTransportStreamVisitor; |
+| 1 | Application calls the factory method of WebTransportFactory::Create() to get an instance of WebTransportFactory |
+| 2 | Application calls WebTransportFactory::CreateWebTransportServer() on the WebTransportFactory instance got in step #1, specifying the server port, certificate path either in the form of .pkcs12 or .pfx format. |
+| 3 | Application Creates the Visitor instance of WebTransportServerInterface |
+| 4 | Application calls SetVisitor on the WebTransportServerInterface instance got in step #2. |
+| 5 | Application calls Start() method on the WebTransportServerInterface instance got in step #2 |
+| 6 | OnSession() callback will be invoked once the WebTransportserver gets a connection request from client, and an WebTransportSession instance is passed from the callback. |
+| 7 | Application calls CreateBidirectionalStream on the WebTransportSession got in step 6 and get a WebTransportStreamInterface instance. |
+| 8 | Application creates WebTransportStream::Visitor instance and invokes WebTransportStreamInterface::SetVisitor(). |
+| 9 | Application reads the WebTransportStream when OnCanRead is invoked on the WebTransportStream::Visitor; or write to the WebTransportStream when OnCanWrite is invoked on the WebTransportStreamVisitor; |
 
 ## OWT QUIC C++ SDK Client API Calling Flow
 
-The client API calling flow is shown in below diagram and table. It's similar as the server side calling flow except the QuicTransportFactory creates a QuicTransportClientInterface, instead of a QUICTransportServerInterface, and client needs to call Connect() instead of Start() to get a QuicTransportSession.
+The client API calling flow is shown in below diagram and table. It's similar as the server side calling flow except the WebTransportFactory creates a WebTransportClientInterface, instead of a WebTransportServerInterface, and client needs to call Connect() instead of Start() to get a WebTransportSession.
 
 ![plot](./pics/client_API.png)
```

Comment:
I‘m not sure if the picture needs to be updated as well?

Reply:
Yes, class names need to be updated. But they're pngs instead of svgs, so i'm not able to update them.

Reply:
Updated.

Reply:
Block diagram is changed significantly. Link between two native clients is removed. please correct me if it's wrong.